k8s/upgrades/0.8.0-0.8.1/jiva-target-patch.tpl.json

```diff
@@ -0,0 +1,31 @@
+{
+   "metadata": {
+      "annotations": {
+         "openebs.io/storage-class-ref": "name: @sc_name\nresourceVersion: @sc_resource_version\n"
```

Comment:
I'm seeing prometheus annotations in previous scripts.. will that be needed? (or) CAS is taking care of them already

Reply:
ok.. got to see this in runtask.. so we are good with prometheus annotations

Reply:
CAS will be taking care of them